pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/MultiStageOperator.java

```diff
@@ -39,21 +35,18 @@ public abstract class MultiStageOperator implements Operator<TransferableBlock>,
 
   // TODO: Move to OperatorContext class.
   protected final OperatorStats _operatorStats;
```

Comment:
do we still need this as final reference? can we also put this in OpChainStats and access via OpChainStats?

Reply:
Yes, we still need it as OperatorStats contains stats only for a single operator.

Reply:
IIUC, there will be 0 or 1 OperatorStats object corresponding to this operator in the OpChainStats map. if so why do we keep strong reference to both objects in 2 classes?

Reply:
Cool. So now I have made the change to use directly via the map in OpChainStats rather than creating and storing a strong reference in  the base class.